commitlintplugins.test.ts

```diff
@@ -138,6 +138,27 @@ test('body-max-line-length6', () => {
 });
 
 
+test('empty-wip-1', () => {
+    let commitMsgWithEpmtyWIP = "WIP";
+    let emptyWip = runCommitLintOnMsg(commitMsgWithEpmtyWIP);
+    expect(emptyWip.status).not.toBe(0);
+});
+
+
+test('empty-wip-2', () => {
+    let commitMsgWithDescriptionAfterWIP = "WIP: bla bla blah";
+    let wipWithDescription = runCommitLintOnMsg(commitMsgWithDescriptionAfterWIP);
+    expect(wipWithDescription.status).toBe(0);
+});
+
+
+test('empty-wip-3', () => {
+    let commitMsgWithNumberAfterWIP = "WIP1";
+    let wipWithNumber3 = runCommitLintOnMsg(commitMsgWithNumberAfterWIP);
```

Comment:
vars in lines 143, 150 and 157 have to be renamed (pls follow same format as other tests)

Reply:
s/resolve/address

Reply:
sorry, it was an --amend problem